debian/control

```diff
@@ -13,6 +13,7 @@ Build-Depends: asciidoc (>= 8.6.6),
                libboost-system-dev,
                libboost-program-options-dev,
                python3,
+	       libyaml-cpp-dev
```

Comment:
something looks wrong with _whitespace_ indentation here.

Reply:
Seems my editor has mixed spaces and tabs for some reason